src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -70,7 +70,16 @@ public static partial class SymbolFinder
 
             var semanticInfo = await GetSemanticInfoAtPositionAsync(
                 semanticModel, position, services, cancellationToken).ConfigureAwait(false);
-            return semanticInfo.GetAnySymbol(includeType: false);
+            var symbol = semanticInfo.GetAnySymbol(includeType: false);
```

Comment:
i don't understand why GetSmeanticInfoAtPosition doesn't just return the pp symbol.

Reply:
Because that normally returns a `TokenSemanticInfo`, based on which I determine whether we found a symbol

Reply:
i don't understand the answer.  why can't we return the PP information in the TokenSemanticInfo?

Reply:
`GetSemanticInfo` does not return pp symbols. Do we want to make the change there, or specifically construct a new TokenSemanticInfo from the PreprocessingSymbolInfo that we get from the other method?

Reply:
> GetSemanticInfo does not return pp symbols

We own GetSemanticInfo.  It's purpose is to say "what is the semantic info of the thing at this position".  It is used by these other features so they don't need special logic.  By adding special logic to them, it's defeating the purpose :)

Reply:
GetSemnaticInfo now returns pp symbols, just noting here in case of additional comments on the implementation